soc/raspberrypi/rpi_pico/rp2350/Kconfig

```
@@ -9,6 +9,22 @@ config SOC_SERIES_RP2350
 	select SOC_RESET_HOOK
 	select XIP
 
+config SOC_RP2350A_HAZARD3
+	select HAS_FLASH_LOAD_OFFSET
+	select INCLUDE_RESET_VECTOR
+	select RISCV
+	select RISCV_ISA_RV64I
```

Why do we select RV64I here? Rust builds fail for RV64I, RV32I is okay (the CPU is 32-bit)
```suggestion
	select RISCV_ISA_RV32I
```

---

resolved